tasks/Linux/selinux-support.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: "Install ansible selinux support library"
+  package:
+    name: libselinux-python
+    state: present
+  become: True
+
+- name: "Install ansible selinux configure libraries"
+  package:
+    name: "{{ selibs_item }}"
+    state: present
+  with_items:
+    - policycoreutils-python
+    - libsemanage-python
+  loop_control:
+    loop_var: selibs_item
+  become: True
+  when: ansible_selinux.status == "enabled"
```

Comment:
Package operations are unreliable as they require network communication and the availability of remote servers. To mitigate the potential problems, retries should be used via ``register: my_result`` and ``until: my_result | success``

Reply:
right, fixed